package.json

```diff
@@ -10,7 +10,7 @@
     "css": "npm-run-all css-compile css-prefix css-minify",
     "css-compile": "sass --style expanded --source-map --embed-sources --no-error-css scss/:dist/css/",
     "css-lint": "npm-run-all --aggregate-output --continue-on-error --parallel css-lint-*",
-    "css-lint-stylelint": "stylelint \"**/*.{css,scss}\" --cache --cache-location .cache/.stylelintcache",
+    "css-lint-stylelint": "stylelint \"**/*.{css,scss}\" --fix --cache --cache-location .cache/.stylelintcache",
```

Comment:
```suggestion
    "css-lint-stylelint": "stylelint \"**/*.{css,scss}\" --cache --cache-location .cache/.stylelintcache",
```

Reply:
Do we trust stylelint to always do the right thing?